photon-client/src/main/scala/com/linkedin/photon/ml/Constants.scala

```diff
@@ -26,16 +26,16 @@ object Constants {
    *
    * WARNING: This is not visible in println!
    */
-  val DELIMITER = "\u0001"
+  val DELIMITER: String = "\u0001"
```

Comment:
I don't think explicit types are necessary here, it's clear that all of these are `String`